scout/server/blueprints/alignviewers/templates/alignviewers/igv_viewer.html

```diff
@@ -70,6 +93,31 @@
                         height: "{{track.height}}"
                       },
                       {% endfor %}
+                      {% for ttrack in tiddit_wig_tracks %}
+                      {
+                        type: "wig",
+                        name: '{{ ttrack.name }}',
+                        url: '{{ url_for("alignviewers.remote_static", file=ttrack.url) }}',
+                        format: 'bigwig',
```

Comment:
Same here - wig is produced right now. This may not be ideal, but lets make it wig to allow for a live test.